server/settings.py

```diff
@@ -162,6 +162,7 @@ def env(variable, fallback_value=None):
     'superdesk.upload',
     'superdesk.notification',
     'superdesk.activity',
+    'superdesk.vocabularies',
```

Comment:
Any specific reason for registering `superdesk.vocabularies` in both `superdesk` and `superdesk-core`? Ideally it should be registered in `superdesk-core` only.